relnotes/array_bcontains.feature.md

```diff
@@ -0,0 +1,8 @@
+### Add bcontains which uses bsearch
```

Comment:
In cases like that I think it is more convenient for users of ocean to have one `### New array search utilities` entry which lists all added functions